sql/executor.go

```diff
@@ -345,3 +346,14 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
 		panic(fmt.Sprintf("unexpected type %T", t))
 	}
 }
+
+func (s Session) getLocation() (*time.Location, error) {
```

Comment:
Minor nit: our convention so far has been to place methods on protos in a file with the same base name as the `.proto`. So this should go in `session.go`.